### ISSUE TITLE
Can't import pug file from js - error - Cannot find module

### ISSUE THREAD

**Issue by pbastowski** [parcel-bundler/parcel#1751]
# 🐛 bug report

I am trying to import a pug file in js, as shown below. This code works in webpack (with pug and pug-plain-loader installed), but fails with Parcel.

```js
import template from './template2.pug'
console.log('html:', template)
```

If you remove the `console.log` then the error disappears and nothing happens.

## 🎛 Configuration (.babelrc, package.json, cli command)

Config from package.json
```js
  "dependencies": {
    "@uirouter/angularjs": "^1.0.18",
    "angular": "^1.7.2",
    "ng2now": "^0.0.17"
  },
  "devDependencies": {
    "node-sass": "^4.9.0",
    "parcel-bundler": "1.9.7",
    "parcel-plugin-markdown": "^0.3.1",
    "pug": "^2.0.3",
    "typescript": "^2.9.2"
  }
```

## 🤔 Expected Behavior

The `template` variable contains the HTML compiled by PUG (from `./template2.pug`)

## 😯 Current Behavior

I get en error
```
Uncaught (in promise) Error: Cannot find module 'src/tasks/template2.jade'
    at newRequire (src.299c1486.js:39)
    at newRequire (src.299c1486.js:23)
    at localRequire (src.299c1486.js:54)
    at Object.parcelRequire.src/tasks/index.ts.ng2now (index.ts:5)
    at newRequire (src.299c1486.js:48)
    at localRequire (src.299c1486.js:54)
    at Object.parcelRequire.src/app/index.ts.ng2now (index.ts:3)
    at newRequire (src.299c1486.js:48)
    at localRequire (src.299c1486.js:54)
    at Object.parcelRequire.src/index.ts.angular (index.ts:6)
```

## 💁 Possible Solution

Don't know.

## 🔦 Context

Can't import pug files in js.

## 💻 Code Sample

No need for any complex repo, the code below fails, as explained above, even with the latest version 1.9.7

**index.js**
```js
import template from './template2.pug'
console.log('html:', template)
```

**template.pug**
```pug
h2 This is a PUG file
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.9.7
| Node             | 9.11.1
| npm/Yarn         | yarn 1.7
| Operating System | MacOS 10.13.6

**Comment by DeMoorJasper** [comment]
Pug gets transpiled into html so this is actually fixed by https://github.com/parcel-bundler/parcel/commit/075ca6fe734fddb735701be0df04375d754a0cdd

**Comment by pbastowski** [comment]
Thanks for the quick response. Is this fix included in v1.9.7?
